src/js/datecalc.js

```diff
@@ -0,0 +1,20 @@
+// import { DateTime } from "https://moment.github.io/luxon/es6/luxon.min.js";
+import { DateTime } from "../../node_modules/luxon/src/luxon.js"
```

Comment:
тут можно попроще написать, вебпак и так по умолчанию ищет зависимости в node_modules
```suggestion
import { DateTime } from "luxon"
```